macros/helpers/filament_swap.cfg

```diff
@@ -2,11 +2,12 @@
 description: Do a PAUSE, park the toolhead over the purge bucket and unload the filament
 gcode:
     {% set TEMP = params.TEMP|default(printer["gcode_macro _USER_VARIABLES"].print_default_extruder_temp)|float %}
+    {% set DISTANCE = params.DISTANCE|default(105)|float %}
```

Comment:
not sure if this was in the mmu PR, but would it be possible to add this is variable in variables.cfg in case users want to have a different value?